apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+import { Feature as FeatureEC } from '@pagopa/pagopa-editorial-components';
+import { FeatureProps } from '@pagopa/pagopa-editorial-components/dist/components/Feature/Feature';
+import { Icon } from '@mui/material';
+import { formatValidMuiIcon, isValidMuiIcon } from './Icons';
+import { FeatureSection } from '@/lib/fetch/types/PageSection';
+
+const makeFeatureProps = ({
+  items,
+  background,
+  ...rest
+}: FeatureSection): FeatureProps => ({
+  items: items.map((item) => ({
+    stackIcon: {
+      ...(isValidMuiIcon(item.icon) && {
+        startIcon: <Icon>{formatValidMuiIcon(item.icon)}</Icon>,
+      }),
+      color: item.iconColor,
+    },
+    title: item.title,
+    subtitle: item.subtitle,
+    link: { text: item.linkText || '', url: item.linkURL || '' },
```

Comment:
Instead of putting empty values, I would prefer to avoid setting the property.

```suggestion
    ...(item.linkText &&
      item.linkURL && { link: { text: item.linkText, url: item.linkURL } }),
```